src/main/java/picard/util/IntervalListTools.java

```diff
@@ -495,6 +514,22 @@ private IntervalList openIntervalLists(final List<File> files, BinaryOperator<In
                 .orElse(null);
     }
 
+    private static IntervalList uniqued(IntervalList non_unique, boolean merge_abutting) {
+        // A subroutine to replace htsjdk's IntervalList "unique()" method, which combines abutting intervals by default.
+        // Returns an IntervalList.
+        // Inputs:
+        //  - non_unique: an IntervalList to be operated on
+        //  - merge_abutting: a boolean to combine abutting intervals (if true) or not (if false).
+        // Outputs:
+        //  - an IntervalList that is uniqued with overlapping intervals merged, and abutting intervals handled per the boolean flag.
+        final boolean CONCATENATE_NAMES = true;
```

Comment:
why are these in variables? you can just put `true,false` in the method call, no?

Reply:
I definitely can, it just makes for more readable code. Purely design decision, happy to change it

Reply:
OK. So in that case, just change the variable names to camelCase

Reply:
done 👍 :)